client/app/dedicatedCloud/security/kms/add/dedicatedCloud-security-kms-add.html

```diff
@@ -0,0 +1,116 @@
+                       }">
+                <input class="oui-input"
+                       type="text"
+                       id="sslThumbprint"
+                       name="sslThumbprint"
+                       data-ng-model="$ctrl.kms.sslThumbprint"
+                       data-ng-pattern="$ctrl.regex.sslThumbprint"
+                       data-required>
+            </oui-field>
+            <oui-checkbox data-model="$ctrl.documentationRead">
+                <span data-translate="dedicatedCloud_vm_encryption_add_kms_confirm_read_documentation"></span>
+            </oui-checkbox>
+            <a class="oui-link oui-link_icon ml-4 pl-3"
+               data-href=""
+               target="_blank"
+               rel="noopener">
+                <span data-translate="dedicatedCloud_vm_encryption_kms_documentation_link"
+                      aria-hidden="true"></span>
+                <span class="oui-icon oui-icon-external_link"></span>
+            </a>
+        </form>
+    </div>
+
+    <div data-wizard-step
+         data-wizard-step-on-load="$ctrl.createNewKms.bind($ctrl)"
+         data-wizard-step-valid="$ctrl.isSecondStepValid()">
+        <label for="kmsCreationProgress"
+               class="oui-label">
+            <strong data-translate="dedicatedCloud_vm_encryption_add_kms_current_operation"
+                    data-translate-values="{ 'currentOperation': $ctrl.kmsCreationTask.name }"></strong>
+        </label>
+        <oui-progress class="mb-4" id="kmsCreationProgress">
+            <oui-progress-bar data-type="info"
+                              data-value="$ctrl.kmsCreationTask.progress">
+                <span data-ng-bind="$ctrl.kmsCreationTask.description"></span>
+            </oui-progress-bar>
+        </oui-progress>
+        <span data-ng-if="$ctrl.error && $ctrl.error.data"
+              data-translate="dedicatedCloud_vm_encryption_kms_add_error"
+              data-translate-values="{ 'error' : $ctrl.error.data.message }">
+        </span>
+    </div>
+
+    <div data-wizard-step
+         data-wizard-step-valid="$ctrl.isThirdStepValid()">
+        <p class="oui-paragraph">
+            <p data-translate="dedicatedCloud_vm_encryption_add_kms_configuration_intro_1"></p>
+            <p data-translate="dedicatedCloud_vm_encryption_add_kms_configuration_intro_2"></p>
+        </p>
+        <p class="oui-paragraph">
+            <p data-translate="dedicatedCloud_vm_encryption_add_kms_configuration_intro_3"></p>
+            <a class="oui-link oui-link_icon"
+               data-href=""
+               target="_blank">
+                <span data-translate="dedicatedCloud_vm_encryption_kms_documentation_link"
+                      aria-hidden="true"></span>
+                <span class="oui-icon oui-icon-external_link"></span>
+            </a>
+        </p>
+    </div>
+
+    <div data-wizard-step
+         data-wizard-step-valid="$ctrl.isFinalStepValid()">
+        <label for="kmsCreationProgress"
```

Comment:
`for` is supposed to point to the `id` of a `labellable` element. `oui-progress` is not labellable. You want to use `aria-describedby`